samples/subsys/usb_c/sink/boards/b_g474e_dpow1.overlay

```diff
@@ -27,8 +27,28 @@
 			tcpc = <&ucpd1>;
 			vbus = <&vbus1>;
 			power-role = "sink";
+            		num-snk-pdos= <1>;
 			sink-pdos = <PDO_FIXED(5000, 100, 0)>;
+            		fr-swap-reqd-type-c-current-as-initial-source= <0>;
```

Comment:
missing space before `=`, please revisit formatting on all files.

Reply:
Included the changes in revision of this PR